lib/config.js

```diff
@@ -37,7 +35,8 @@ export default class Config {
         jade: Joi.string().default('**/*.jade'),
         css: Joi.string().default('**/*.+(css|sss)'),
         js: Joi.string().default('**/*.js'),
-        static: Joi.string().default(`**/*.{${union(binaryExtensions, imageExtensions).join(',')}}`)
+        // this has to not match any webpack entry file
+        static: Joi.string().default('!**/*.+(js|css|sss|jade)')
```

Comment:
yeah, i agree. i figured it was worth discussing, since we hadn't yet, but we'll just end up with `.sss` in our roots-mini-base and most people will use it, but they won't **have** to